### PR TITLE
Apps

### PR DESCRIPTION
**Pull request by Adam-Goldman7**

### CODE REVIEW

File: client/containers/AppListPage/index.js [hunk 1]
```diff
@@ -18,21 +18,27 @@ class AppListPage extends Component {
...
         apps: data
       }));
   }
 
   renderApps () {
-    if (!this.state.apps) {
-      return;
+    if (!this.state.apps.length) {
+      // TODO [adgo] 26.04.2016 - change to real link
+      return <h3>No apps do diplay. Go ahead and <a href='#'>add one</a>!</h3>;
```
Comment (tomusdrw) [inline client/containers/AppListPage/index.js]: typos in description

File: client/containers/AppListPage/index.js [hunk 2]
```diff
@@ -18,21 +18,27 @@ class AppListPage extends Component {
...
     }
-    return this.state.apps.map((app) => (
-      <li key={app}>
-        <a href={`/${app}/`}>{app}</a>
-      </li>
-    ));
+
+    return this.state.apps.map((app) => {
+      const href = `//${window.location.hostname}:8080/${app}/`;
```
Comment (tomusdrw) [inline client/containers/AppListPage/index.js]: webapp port is configurable. Just use `window.location.host`, but actually `/${app}/` should be sufficient.

Comment (Adam-Goldman7) [inline client/containers/AppListPage/index.js]: Without explicitly setting the port it doesnt work on testing and dev

Comment (tomusdrw) [inline client/containers/AppListPage/index.js]: It doesn't have to work in dev. In parity there is CLI option `--webapp-port` that allows you to change that port - no way to figure that out in frontend.

**Pull request merged**